sensing/gnss_poser/include/gnss_poser/convert.hpp

```diff
@@ -99,6 +99,41 @@ GNSSStat NavSatFix2UTM(
   }
   return utm;
 }
+bool is_utm_origin_initialized = false;
+GNSSStat utm_origin;
+GNSSStat utm_local;
+GNSSStat NavSatFix2LocalCartesianUTM(
+  const sensor_msgs::msg::NavSatFix & nav_sat_fix_msg,
+  sensor_msgs::msg::NavSatFix nav_sat_fix_origin_, const rclcpp::Logger & logger)
+{
+  utm_origin.coordinate_system = CoordinateSystem::UTM;
+  utm_local.coordinate_system = CoordinateSystem::UTM;
+  double global_x;
+  double global_y;
+  try {
+    if (is_utm_origin_initialized == 0) {
+      GeographicLib::UTMUPS::Forward(
+        nav_sat_fix_origin_.latitude, nav_sat_fix_origin_.longitude, utm_origin.zone,
+        utm_origin.northup, utm_origin.x, utm_origin.y);
+      utm_origin.z = EllipsoidHeight2OrthometricHeight(nav_sat_fix_msg, logger);
+      is_utm_origin_initialized = true;
+    } else {
```

Comment:
Do we need `else` here?
I believe we still want to do the projection even for the first time this function is called.

Reply:
I tried to specify an origin to the local coordinate system. Also we do the projection to all messages that received. "if" part projects the origin.

Reply:
You're right. There is no need to `else` here